examples/jupyter/plot_DOFs_and_qpts.ipynb

```diff
@@ -0,0 +1,374 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DOF and quadrature point plotter\n",
+    "\n",
+    "This notebook helps visualize the locations of MFEM's DOFs and quadrature points for various element types and quadrature rules.\n",
+    "\n",
+    "To use this, your python environment must have:\n",
+    "* PyMFEM (https://github.com/mfem/PyMFEM)\n",
+    "* Matplotlib (https://matplotlib.org)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preamble: import any necessary libs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mfem.ser as mfem\n",
+    "\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import math\n",
+    "\n",
+    "%matplotlib inline\n",
```

Comment:
(minor) not needed since you plot to files, did you want to have an option at the top like `save_to_files` that is `True` by default?

Reply:
Not sure about this one -- this notebook both plots the generated figures and saves them to disk.
Are you suggesting that it should be converted to one or the other?